.github/workflows/builder_nodejs_slsa3.yml

```diff
@@ -220,62 +218,64 @@ jobs:
           sha256: "${{ needs.builder.outputs.node-builder-sha256 }}"
           set-executable: true
 
-      - name: Download dependencies
+      # TODO(hermeticity)
+      - name: Build project
         working-directory: __PROJECT_CHECKOUT_DIR__
         env:
-          UNTRUSTED_CI_ARGUMENTS: "${{ inputs.ci-arguments }}"
-          UNTRUSTED_DIR: ${{ inputs.directory }}"
+          SLSA_WORKFLOW_INPUTS: "${{ inputs }}"
         run: |
           set -euo pipefail
 
-          # npm ci <args>
-          "$GITHUB_WORKSPACE/$BUILDER_BINARY" ci \
-            --ci-arguments "$UNTRUSTED_CI_ARGUMENTS" \
-            --directory "$UNTRUSTED_DIR"
+          "$GITHUB_WORKSPACE/$BUILDER_BINARY" build
 
-      # TODO(hermeticity) Enable OS-level hermeticity.
+      - name: Upload generated tarball
+        id: upload
+        uses: ./__BUILDER_CHECKOUT_DIR__/.github/actions/secure-upload-artifact
+        with:
+          name: "<TODO>"
+          path: "<TODO>"
```

Comment:
Were you planning on setting these?

Reply:
not in this PR. The output of the builder can theoretically contain multiple artifacts, so I think we need to write code to process a number of artifacts at once. For nodejs, I think it will always be a single artifact. But in general I'm trying to se what's needed to build a "generic workflow" that generalizes for any builder (nodejs, Java, goreleaser, etc)